config/routes.rb

```diff
@@ -8,6 +8,7 @@
       devise_scope :user do
         post 'auth/facebook', to: 'sessions#facebook'
       end
+      resources :users, only: [:update]
```

Comment:
This is just fine, you can try to use the `%i()` syntax. `%i(update)`.
[See more here](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/SymbolArray)